examples/basic_dispatcher.py

```diff
@@ -1,33 +1,60 @@
 # SPDX-License-Identifier: MIT
 
-# This example uses disnake
+# Basic functionality example using disnake.
+
+import os
 
 import disnake
 from disnake.ext import commands
 
 from dispatcher import Dispatcher
 
-bot = commands.Bot(command_prefix=commands.when_mentioned, intents=None)
-bot.dispatcher = Dispatcher(bot, library_name=disnake.__name__)
 
+# We don't need prefix commands, so we are using InteractionBot here.
+class MyBot(commands.InteractionBot):
+    # Dispatcher uses generics so that you can use any bot
+    # class from any library: you just need to type-hint it.
+    dispatcher: Dispatcher["MyBot"]
+
+    def __init__(self) -> None:
+        super().__init__(intents=disnake.Intents.default())
+        # One would usually assign dispatcher to the bot directly.
+        self.dispatcher = Dispatcher(self)
+
+    async def on_ready(self) -> None:
+        print(f"Ready! Logged in as {self.user}")
+
+        # Send message to any available channel.
+        for guild in self.guilds:
+            for channel in guild.text_channels:
+                await channel.send("Hello!")
+                break
+            break
```

Comment:
idk what's the purpouse of this, 
>  that's an example intended to showcase the extension's possibilities

but it isn't showing a specifically action that this extension makes possible, furthermore if we want to mantain it we should mention with comments that making API call inside an on_ready event is higly discouraged for bla bla...

Reply:
Alr i'll imagine a better example later

Reply:
dc8c2333f909f3c01f1313fb24a6b6cb997b70bc